tests/plotting/test_plotting.py

```diff
@@ -2468,6 +2468,20 @@ def test_add_text():
     plotter.show()
 
 
+@pytest.mark.skipif(
+    vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported(),
+    reason='Math text is not supported.',
```

Comment:
Is it on purpose that a call to `MathTextIsSupported` gives us whether math text is _not_ supported?

Reply:
🤦🏻 Thank you for catching this... this is what happens when I code at night.

Reply:
It looks like the conda build of VTK is giving `True` for `vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported()` even though it isn't supported 😕 

https://github.com/conda-forge/vtk-feedstock/issues/79